js/filters.js

```diff
@@ -117,7 +117,9 @@ weechat.filter('DOMfilter', ['$filter', '$sce', function($filter, $sce) {
                 }
             }
             // recurse
-            if (node === undefined || node === null) return;
+            if (node === undefined || node === null) {
+                return;
+            }
```

Comment:
Yes, it's more cumbersome. But it's easier to maintain, and easier to stick to one style and follow it. Less chance of introducing bugs later.